docs/src/howto/install/snap.md

```diff
@@ -0,0 +1,69 @@
+# Install Canonical Kubernetes from a snap
+
+Canonical Kubernetes is packaged as a [snap], available from the 
+snap store for all supported platforms.
+
+## What you'll need
+
+This guide assumes the following:
+
+- You are installing on Ubuntu 22.04 or later, **or** another OS which supports
+  snap packages (see [snapd support])
+- You have root or sudo access to the machine
+- You have an internet connection
+- The target machine has sufficient memory and disk space. To accommodate
+  workloads, we recommend a system with at least 20G of disk space and 4G of
+  memory.
+
+```{note}
+If you cannot meet these requirements, please see the [Installing] page for alternative options.
```

Comment:
```suggestion
If you cannot meet these requirements, please see the [Installing][installing] page for alternative options.
```

Reply:
I'm pretty sure this link works - I think it just doesn't render properly in GH

Reply:
ack, gh preview didn't so i thought it was the different case. i trust you may have been right, but appreciate the update to the links section anyway :)